Documentation/Signoffs/sensor_subsystem.md

```diff
@@ -0,0 +1,164 @@
+
+# Sensor Subsystem
+
+## **Function:**
+
+The goal of this subsystem is to accurately and quickly gather several data points to determine the
+distance, speed, and position of the target as it slides down the fishing line.
+
+## **Constraints:**
+
+| No. | Constraint                                                                                              | Origin            |
+| --- | ------------------------------------------------------------------------------------------------------- | ----------------- |
+| 1   | The sensor shall be supplied 5 V via USB from the Jetson Nano processor                                 | System Constraint |
+| 2   | The sensor shall be able to retrieve at least 2 data points within 1.95 s in order to calculate speed   | System Constraint |
+| 3   | The sensor shall have a resolution great enough so that the golf ball spans multiple pixels from 6 feet | Conceptual Design |
+| 4   | The sensor shall have a range of at least 6 feet                                                        | Conceptual Design |
+| 5   | The sensor shall be able to detect and track a golf ball-sized object from a maximum of 6 feet          | System Constraint |
+| 6   | The sensor shall have a field of view greater than 24.50°                                               | Device Constraint |
```

Comment:
This constraint's reasoning does not suggest that this is correct. You aren't placing the sensor at the point of convergence. Therefore, the angle of the strings tells you little about this system in isolation.